src/sinks/sematext/logs.rs

```diff
@@ -24,14 +24,21 @@ use crate::{
 #[configurable_component(sink("sematext_logs"))]
 #[derive(Clone, Debug)]
 pub struct SematextLogsConfig {
+    #[serde(default = "super::default_region")]
     #[configurable(derived)]
-    region: Option<Region>,
+    region: Region,
 
     /// The endpoint to send data to.
+    ///
+    /// Setting this option overrides the `region`.
```

Comment:
nit nit nit: "This option overrides `region`." , or "Setting this option overrides the `region` option" is more clear imo. I find it slightly confusing wording to use "setting" and "option" in the same sentence when "option" could be interchanged with "setting" 🙄

Reply:
Resolved in [cd5b1b8](https://github.com/vectordotdev/vector/pull/16405/commits/cd5b1b805ded4ca84971e261e3c7f442970f3a2d)

Definitely see what you're saying with `option` vs `setting` - I'll think about that.